lib/lang/vi.js

```diff
@@ -0,0 +1,167 @@
+  "tomorrow-night": "tối mai",
+  "morning": "vào buổi sáng",
+  "afternoon": "vào buổi chiều",
+  "evening": "vào buổi tối",
+  "night": "đêm",
+  "today": "hôm nay",
+  "tomorrow": "vào ngày mai",
+  "sunday": "vào chủ nhật",
+  "monday": "vào thứ hai",
+  "tuesday": "vào thứ ba",
+  "wednesday": "vào thứ tư",
+  "thursday": "vào thứ năm",
+  "friday": "vào thứ sáu",
+  "saturday": "vào thứ bảy",
+  "next-sunday": "chủ nhật tuần sau",
+  "next-monday": "thứ hai tuần sau",
+  "next-tuesday": "thứ ba tuần sau",
+  "next-wednesday": "thứ tư tuần sau",
+  "next-thursday": "thứ năm tuần sau",
+  "next-friday": "thứ sáu tuần sau",
+  "next-saturday": "thứ bảy tuần sau",
+  "minutes": "$1 phút",
+  "fahrenheit": "$1\u00B0F",
+  "celsius": "$1\u00B0C",
+  "inches": "$1 in",
+  "centimeters": "$1 cm",
+  "less-than": "dưới $1",
+  "and": function(a, b) {
+    return join_with_shared_prefix(
+      a,
+      b,
+      a.indexOf(",") !== -1 ? ", và " : " và "
+    );
+  },
+  "through": function(a, b) {
+    return join_with_shared_prefix(a, b, " cho tới ");
+  },
+  "with": "$1, với $2",
+  "range": "$1\u2013$2",
+  "parenthetical": function(a, b) {
+    return a + " (" + b + (a === "mixed precipitation" ? " of snow)" : ")");
+  },
+  "for-hour": "$1 trong một giờ",
+  "starting-in": "$1 bắt đầu sau $2",
+  "stopping-in": "$1 dừng sau $2",
+  "starting-then-stopping-later": "$1 bắt đầu sau $2, dừng lại $3 sau",
+  "stopping-then-starting-later": "$1 dừng sau $2, tiếp tục $3 sau",
+  "for-day": "$1 cả ngày",
+  "starting": "$1 bắt đầu lúc $2",
+  "until": function(condition, period) {
+    return condition + " tới " + strip_prefix(period);
+  },
+  "until-starting-again": function(condition, a, b) {
+    return condition + " cho đến " + strip_prefix(a) + ", bắt đầu lại " + b;
+  },
+  "starting-continuing-until": function(condition, a, b) {
+    return condition + " bắt đầu lúc " + a + ", tiếp tục tới khi " +
+           strip_prefix(b);
+  },
+  "during": "$1 $2",
+  "for-week": "$1 cả tuần",
+  "over-weekend": "$1 suốt cuốt tuần",
+  "temperatures-peaking": "nhiệt độ đỉnh điểm $1 $2",
+  "temperatures-rising": "nhiệt độ tăng tới $1 $2",
+  "temperatures-valleying": "nhiệt độ thấp nhất $1 $2",
+  "temperatures-falling": "nhiệt độ giảm tới $1 $2",
+  // Capitalize the first letter of every word, except if that word is
+  // "and". (This is a very crude bastardization of proper English titling
+  // rules, but it is adequate for the purposes of this module.)
+  "title": function(str) {
+    return str.replace(
+      /\b(?:a(?!nd\b)|c(?!m\.)|i(?!n\.)|[^\Waci])/g,
+      function(letter) {
+        return letter;
+      }
+    );
+  },
+  // Capitalize the first word of the sentence and end with a period.
+  "sentence": function(str) {
+    // Capitalize.
+    str = str.charAt(0).toUpperCase() + str.slice(1);
+
+    // Add a period if there isn't already one.
+    if(str.charAt(str.length - 1) !== ".")
+      str += ".";
+
+    return str;
+  },
```

Comment:
Is this the correct capitalization rule for Vietnamese?

Reply:
Yes, this is  the correct capitalization rule for Vietnamese